docs/running-rules.md

```diff
@@ -9,39 +9,39 @@ import MoreHelp from "/src/components/MoreHelp"
 
 ## Introduction
 
-Rules are specific patterns based on which Semgrep reports findings in code. These findings may help you to catch issues of security, performance, correctness, and other bugs. Rules are stored in open-source Semgrep Registry that enables you to scan code without need to write anything custom.
+Rules are specific patterns based on which Semgrep reports findings in code. These findings may help you to catch issues of security, performance, correctness, and other bugs. Rules are stored in an open-source [Semgrep Registry](https://github.com/returntocorp/semgrep-rules) that enables you to scan code without need to write anything custom. The list below covers different kinds of Semgrep rules:
```

Comment:
"Rules are specific patterns based on which Semgrep reports findings in code"

I think this first sentence is already hard to read and may need a rewrite. 

Maybe something like:
"Semgrep reports findings in code based on rules, which use specific patterns to match code."

Reply:
Sure, works for me 👍 

Minor grammar error to fix:

> Rules are stored in an open-source [Semgrep Registry](https://github.com/returntocorp/semgrep-rules) that enables you to scan code without need to write anything custom.

Change to:
Rules are stored in an open-source [Semgrep Registry](https://github.com/returntocorp/semgrep-rules) that enables you to scan code without the need to write anything custom.

Reply:
Changing, thank you!